utils/mvn/utils.go

```diff
@@ -176,6 +177,20 @@ func setEmptyDeployer(vConfig *viper.Viper) {
 	vConfig.Set(utils.DEPLOYER_PREFIX+utils.SNAPSHOT_REPO, "empty_repo")
 }
 
+func handleIncludeExcludePatterns(vConfig *viper.Viper) {
+	fixPatternsSeparator(vConfig, utils.INCLUDE_PATTERNS)
+	fixPatternsSeparator(vConfig, utils.EXCLUDE_PATTERNS)
+}
+
+// The extractor expects the separator to be ", " while the cli uses ";".
```

Comment:
The ", " separator is actually a good separator.  When I suggested changing it to ";" I was under the impression that the separator expected by the build-info-extractor code is ",=". If this si not the case, let's avoid the unnecessary conversion, and use ", " in the cli code as well.